packages/assets/lib/css.js

```diff
@@ -51,22 +57,23 @@ function cssModules (fileType, options = {}) {
     console.warn(`css.modules(): You are using the deprecated 'fileType' parameter, 'options.exclude' or 'options.include'. Use match() instead.`)
   }
 
-  options = Object.assign({
+  const cssLoader = Object.assign({
     modules: true,
     importLoaders: 1,
     localIdentName: String(process.env.NODE_ENV) === 'production'
-      ? '[hash:base64:10]'
-      : '[name]--[local]--[hash:base64:5]'
-  }, options)
+            ? '[hash:base64:10]'
+            : '[name]--[local]--[hash:base64:5]'
+  }, _.omit(options, ['exclude', 'include', 'styleLoader']))
+  const loaders = [{ loader: 'css-loader', options: cssLoader }]
+  if (options.styleLoader !== false) {
```

Comment:
Is this breaking change intended? Now you actually need to explicitly set `options.styleLoader` to get a `style-loader`.

I would think a vast majority of the users would like to have the style-loader by default and opt-out if necessary.

Reply:
Where do I find that in the code? Don't see it...

Reply:
By default it’s `undefined`, which is !== `false` ;-)

Reply:
Right. It's getting late... ;P

Just wondering if we can somehow make the code a bit more speaking in this particular place.